scilpy/tractanalysis/todi_util.py

```diff
@@ -85,7 +85,13 @@ def streamlines_to_pts_dir_norm(streamlines, n_steps=1, asymmetric=False):
     seg_dir, seg_norm = get_segments_dir_and_norm(segments,
                                                   seg_mid,
                                                   asymmetric)
-    return seg_mid, seg_dir, seg_norm
+
+    mask = seg_norm > 1.0e-20
```

Comment:
is this value big enough to catch anything?

Reply:
Here the only purpose is to make sure norm !=0 otherwise people should use remove_invalid_streamlines with a threshold they can set.

Reply:
Did it at least solve the use case you had (i.e. is it big enough to solve one know case) ?